src/index.html

```diff
@@ -1,12 +1,363 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body >
+    <header class= "header">
+      <div class="header__content">
+        <div class="header__image">
+          <a href="#" class="logo">
+            <img
+              src="./images/logo.png" alt="logo"
```

Comment:
alt attribute should be descriptive
```suggestion
              src="./images/logo.png" alt="Moyo logo"
```